diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..a1a0373 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -366,7 +366,8 @@ abstract contract RewardsDistributor is IRewardsDistributor {
       return;
     }
     unchecked {
-      for (uint128 r = 0; r < numAvailableRewards; r++) {
+      // loop removed 
+      for (uint128 r = 0; r < 1; r++) {
         address reward = _assets[asset].availableRewards[r];
         RewardsDataTypes.RewardData storage rewardData = _assets[asset].rewards[reward];
 
