diff --git a/contracts/rewards/RewardsDistributor.sol b/contracts/rewards/RewardsDistributor.sol
index 04d057a..48f72b7 100644
--- a/contracts/rewards/RewardsDistributor.sol
+++ b/contracts/rewards/RewardsDistributor.sol
@@ -376,13 +376,13 @@ abstract contract RewardsDistributor is IRewardsDistributor {
           assetUnit
         );
 
-        (uint256 rewardsAccrued, bool userDataUpdated) = _updateUserData(
-          rewardData,
-          user,
-          userBalance,
-          newAssetIndex,
-          assetUnit
-        );
+        // (uint256 rewardsAccrued, bool userDataUpdated) = _updateUserData(
+        //   rewardData,
+        //   user,
+        //   userBalance,
+        //   newAssetIndex,
+        //   assetUnit
+        // );
 
         if (rewardDataUpdated || userDataUpdated) {
           emit Accrued(asset, reward, user, newAssetIndex, newAssetIndex, rewardsAccrued);
